added iodine, mirrorrr

bdmorin opened the pull request:
2 useful proxy software

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language```
- [x] Your additions are ordered alphabetically.
- [x] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

danger-awesome-Selfhosted commented:
<!--
  0 Errors
  1 Warning: Found 1 link issue
  0 Messages
  1 Markdown
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Found 1 link issue</td>
    </tr>
  </tbody>
</table>

#### Link issue by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
3 | Dupe | https://github.com/bslatkin/mirrorrr 


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

bdmorin commented:
Where is mirrorrr? I searched, but didn't see it. sorry.

petrk94 commented:
@bdmorin I suppose he mean this: https://github.com/bslatkin/mirrorrr

bdmorin commented:
it's not listed in awesome-sysadmin, nor in awesome-selfhosted. regardless, your call if you want to include it, obviously. heh

Kickball commented on README.md:
```diff
@@ -943,8 +943,10 @@ See https://github.com/n1trux/awesome-sysadmin#metric--metric-collection
+  * [iodine](http://code.kryo.se/iodine/) - A ipv4 over DNS tunnel solution, enabling you to start up a socks5 proxy listener. ([Source Code](https://github.com/yarrick/iodine)) `ISC` `C`
   * [microproxy](https://github.com/thekvs/microproxy) - lightweight non-caching HTTP/HTTPS proxy server. ([Source Code](https://github.com/thekvs/microproxy)) `MIT` `Go`
   * [miniProxy](https://github.com/joshdick/miniProxy/) - Simple web proxy written in PHP that can allow you to bypass Internet content filters, or to browse the internet anonymously. Only one php file. ([Source Code](https://github.com/joshdick/miniProxy/)) `GPLv3` `PHP`
+  * [mirrorrr](https://github.com/bslatkin/mirrorrr) - Python proxy that can run in Google App Engine. ([Source Code](https://github.com/bslatkin/mirrorrr)) `Apache` `Python`
```
* Unable to find Apache License.
* Remove Source Code link if listed in the homepage section (the first link).
* Has not been updated in 8 months.

Please remove it based on the above reasons.

Kickball commented on README.md:
```diff
@@ -943,8 +943,10 @@ See https://github.com/n1trux/awesome-sysadmin#metric--metric-collection
 **[`^        back to top        ^`](#)**
 
   * [http2-serverpush-proxy](https://github.com/n1try/http2-serverpush-proxy) - A reverse proxy that helps to automatically utilize HTTP/2.0's server push mechanism for static websites. Available as middleware and standalone application. `MIT` `Nodejs`
+  * [iodine](http://code.kryo.se/iodine/) - A ipv4 over DNS tunnel solution, enabling you to start up a socks5 proxy listener. ([Source Code](https://github.com/yarrick/iodine)) `ISC` `C`
```
Looks good.

nodiscc commented:
@bdmorin please address the review comments in the following 7 days if you're still interested in having this merged, thanks

bdmorin commented:
Apologies for lack of communications. I updated, removed source code. The project is really useful for hosting a proxy server on Google App Engine, many captive portals allow Google services, and it makes a very easy way to bypass them. It hasn't needed to be updated in a while, as it works great now.

nodiscc commented on README.md:
```diff
@@ -943,8 +944,10 @@ See https://github.com/n1trux/awesome-sysadmin#metric--metric-collection
+  * [iodine](http://code.kryo.se/iodine/) - A ipv4 over DNS tunnel solution, enabling you to start up a socks5 proxy listener. ([Source Code](https://github.com/yarrick/iodine)) `ISC` `C`
   * [microproxy](https://github.com/thekvs/microproxy) - lightweight non-caching HTTP/HTTPS proxy server. ([Source Code](https://github.com/thekvs/microproxy)) `MIT` `Go`
   * [miniProxy](https://github.com/joshdick/miniProxy/) - Simple web proxy written in PHP that can allow you to bypass Internet content filters, or to browse the internet anonymously. Only one php file. ([Source Code](https://github.com/joshdick/miniProxy/)) `GPLv3` `PHP`
+  * [mirrorrr](https://github.com/bslatkin/mirrorrr) - Python proxy that can run in Google App Engine. `Apache` `Python`
```
Please remove mirrorr rit as it is designed to run on Google App Engine, which is not a self-hosted environment.
